apps/docs/content/components/placeholders/Content.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Content
+description: A placeholder for the main content section of a component.
+category: "placeholders"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/layout/src/Content.tsx
+---
+
+<Example src="layout/docs/content/preview" isOpen />
```

Comment:
This is really too thin of a documentation. 

you should add what is in orbiter  like thge description: 

A content placeholder component provides no specific styling by itself, but receives styling from the parent container. In addition, a content placeholder will be automatically placed within the container's layout according to Orbiter guidelines.

Reply:
The thing is that it can't be the description as it'll be too long in the component preview page, and adding this before the example opens the door to having to do the same for all components, which might not ideal.

Reply:
Can you try to figure something out? As of right now we added a page of doc to link to, but the doc page is useless